### PR TITLE
corrected wrong file path & typos

### PR DESCRIPTION
**Pull request by theekrystallee**
updated directory under **_Generating contract types_** to  the correct path:

Screen Shot 2022-11-03 at 3 38 19 AM

### CODE REVIEW

File: src/developer-quickstart.md [hunk 1]
```diff
@@ -24,12 +24,12 @@ The main features of a smart contract that differentiate it from scripts or pred
...
 
-|             | deployable on the blockchain:       | can have state:      | callable on the blockchain: | designed for code reuse: |
-|-------------|-------------------------------------|----------------------|-----------------------------|--------------------------|
-| contract  | ✅                                  | ✅                   | ✅                          | ❌                        |
-| predicate | ✅                                  | ❌                    | ❌                           | ❌                        |
-| script    | ❌                                   | ❌                    | ❌                           | ❌                        |
-| library   | ✅ (via a contract or predicate) | ❌ | ❌                           | ✅                       |
+|           | deployable on the blockchain:    | can have state: | callable on the blockchain: | designed for code reuse: |
+| --------- | -------------------------------- | --------------- | --------------------------- | ------------------------ |
+| contract  | ✅                               | ✅              | ✅                          | ❌                       |
```
Comment (mohammadfawaz) [inline src/developer-quickstart.md:29]: Is the table 100% aligned now in Markdown? Some rows seem a space or two short.

File: src/developer-quickstart.md [hunk 2]
```diff
@@ -147,8 +147,7 @@ impl Counter for Contract {
 }
 ```
 
-> **Note**
-> `return storage.counter;` is equivalent to `storage.counter`.
+> **Note** > `return storage.counter;` is equivalent to `storage.counter`.
```
Comment (mohammadfawaz) [inline src/developer-quickstart.md]: The notes in this file don't all look the same. No strong preference for one form or another, but they should probably be standardized? I personally like `**Note**: ` and keeping the whole note on one line.

**Review by mohammadfawaz — COMMENTED** [review]
Looks great! Thank you so much for this. Just two small comments.

**Review by mohammadfawaz — APPROVED** [review]
I had forgotten to re-review this. Thank you for addressing my comments and for your contribution!

**Comment by theekrystallee** [comment]
thanks for the review and merge. happy to contribute!

**Pull request merged**